akka-sample-kafka-to-sharding-scala/processor/src/main/scala/akka/kafka/KafkaClusterSharding.scala

```diff
@@ -0,0 +1,145 @@
+package akka.kafka
```

Comment:
This would eventually make its way into Alpakka Kafka, but I left it here so it could be easily evolved during the review process.